modin/utils.py

```diff
@@ -617,14 +617,14 @@ def _get_modin_deps_info() -> "dict[str, JSONSerializable]":
 
     try:
         # We import ``DbWorker`` from this module since correct import of ``DbWorker`` itself
-        # from Omnisci is located in it with all the necessary options for dlopen.
-        from modin.experimental.core.execution.native.implementations.omnisci_on_native.db_worker import (  # noqa
+        # from HDK is located in it with all the necessary options for dlopen.
+        from modin.experimental.core.execution.native.implementations.hdk_on_native.db_worker import (  # noqa
             DbWorker,
         )
 
-        result["omniscidbe"] = "present"
```

Comment:
`omniscidbe` is OmniSci package name, so I'd simply use `hdk` instead of `hdkdbe` here and below.